definitions/redux-form_v5.x.x/flow_>=v0.22.1/redux-form_v5.x.x.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React from 'react';
+
+declare module 'redux-form' {
+  declare type InputProps = {
+    name: string,
+    value: string | number | boolean,
+    valid: boolean,
+    invalid: boolean,
+    dirty: boolean,
+    pristine: boolean,
+    active: boolean,
+    touched: boolean,
+    viisted: boolean,
+    autofiller: boolean
+  };
+  declare type FormProps = {
+    active: string,
+    asyncValidate: Function,
+    asyncValidating: string | boolean,
+    destroyForm: Function,
+    dirty: boolean,
+    error: String,
+    fields: { [fieldName: string]: InputProps },
+    handleSubmit: Function,
```

Comment:
`handleSubmit` takes no params and returns `void`, am I reading that right? If so, could we be specific and type this as `() => void`?